atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -222,6 +222,32 @@ def get_remote_urls(self):
 
         return remote_sources_urls
 
+    def get_denylisted_sources(self):
+        src_config = get_source_container(self.workflow, fallback={})
+        denylist_srpms = src_config.get('denylist_srpms')
+        if not denylist_srpms:
+            self.log.debug('denylist_srpms is not defined in reactor_config_map')
+            return []
+
+        denylist_url = denylist_srpms['denylist_url']
+        denylist_key = denylist_srpms['denylist_key']
+        req_session = get_retrying_requests_session()
+
+        response = req_session.get(denylist_url)
+        response.raise_for_status()
+        response_json = response.json()
+
+        if denylist_key not in response_json:
+            raise RuntimeError('Denylist key: {} missing in denylist json from : {}\n{}'.
+                               format(denylist_key, denylist_url, response_json))
+
+        if not isinstance(response_json[denylist_key], list):
```

Comment:
How about content of that list? Should we validate it before passing further?

Reply:
Let's use jsonschema, it will provide better error report (the one implemented in osbs-client)

Reply:
validate what exactly in that list?

Reply:
validation agains schema doesn't provide enough info for users

extended validation in code